.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: bundle install
+
+
```

Comment:
@kkamieniecka 

Thank you, Kate. The other changes are great for the MD files. However, why do we need to add a .gitpod.yaml to the git repo?

Reply:
I think I have removed it now, thanks @khaled196